students/tammyd/lesson02/series.py

```diff
@@ -43,4 +47,46 @@ def sum_series(required, *args, **kwargs):
         for i in range(1,10):
             print(fibonacci(i))
 
-sum_series()
+"""
+
+def sum_series(n, s0=None, s1=None):
+    print("sum_series called:", n)
+    if n == 0:
+        return s0
+    elif n == 1:
+        return s1
+    elif n == 2:
```

Comment:
This was an experiment that failed in class :-(

so be careful -- you only need the first two numbers specified. and they ARE required.